### ISSUE TITLE
Namer function is not supported by ConcurrentRotatingFileHandler

### ISSUE THREAD

**Issue by dashedman** [Preston-Landers/concurrent-log-handler#53]
In accordance with this docs: 

https://python.readthedocs.io/en/stable/library/logging.handlers.html#baserotatinghandler

Standart RotatingHandlers supports usage of custom namer function, to configure file name for rollover process.

could you please implement this logic for ConcurrentRotatingFileHandler?

**Comment by Preston-Landers** [comment]
I agree it makes sense to add that here. I'll look into it when I get a chance. Would be happy to review a PR too.

**Comment by dashedman** [comment]
![image](https://user-images.githubusercontent.com/64865196/229085924-8290114b-ca36-4e4d-8178-ce19fa70a9b6.png)

why I can't create a fork?

**Comment by dashedman** [comment]
#54

**Comment by Preston-Landers** [comment]
Accepted; thanks!
